plugins/feedback-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-feedback-backend",
-  "version": "1.1.3",
+  "version": "1.2.0",
```

Comment:
This version is not published yet? and similar could be the case with other packages

https://www.npmjs.com/package/@janus-idp/backstage-plugin-feedback-backend

Reply:
yes! that way main branch will be x.(y+1).0 while 1.2.x branch is at x.y.z, and can keep doing .z updates that won't collide with the x.(y+1) stuff in main. And main can rev to x.(y+2) if needed as well, as long as it's got a larger y-digit than the 1.yy.x maintenance branches